### PR TITLE
#143 Execute the Syncer in parallel

### PR DESCRIPTION
**Pull request by leungkinghin**
**Current Order**
1. Type/ProductType/State
2. Tax Category
3. Category
4. Product
5. CartDiscount
6. InventoryEntries

**Expected Order**
1. Type/ProductType/State/TaxCategory
2. Category/CartDiscount/InventoryEntries
3. Product

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ dist: trusty
 jdk:
 - openjdk8
 install: true # skips travis' default installation step which executes gradle assemble.
-script: ./gradlew clean build --daemon
```
Comment (lojzatran) [inline .travis.yml:7]: Can you explain why this change?

**Review by lojzatran — APPROVED** [review]
LGTM 👍  Do you see any improvements in the build time?

File: .travis.yml (hunk 1 above)
Comment (leungkinghin) [reply .travis.yml]: Sorry by mistake.
I recovered it.
https://github.com/commercetools/commercetools-project-sync/pull/147/commits/072b19251f6d98a8dd3317b4f26f5d21fa1ba1c2

**Comment by leungkinghin** [comment]
> LGTM 👍 Do you see any improvements in the build time?

Didn't work on time measurement. Instead i input data in source project and run the job in docker manually, to guarantee no missing data when resources is synchronized in parallel.
